ethers-providers/src/rpc/transports/ws/types.rs

```diff
@@ -102,8 +102,8 @@ impl<'de> Deserialize<'de> for PubSubItem {
                                 return Err(de::Error::duplicate_field("error"))
                             }
 
-                            let value: JsonRpcError = map.next_value()?;
-                            error = Some(value);
+                            let value: Option<JsonRpcError> = map.next_value()?;
```

Comment:
this is the part that matters in this file. the rest is just cargo fmt running whenever my editor saves